pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -30,6 +29,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1alpha"
 	"k8s.io/kubernetes/pkg/kubelet/lifecycle"
+	utilstore "k8s.io/kubernetes/pkg/kubelet/util/store"
+	utilfs "k8s.io/kubernetes/pkg/util/filesystem"
```

Comment:
These are new dependencies introduced so you need to run `hack/update-bazel.sh` to refresh the BUILD files.